frontend/src/pages/StudentProfile.jsx

```
@@ -0,0 +1,407 @@
+    };
+    if(user?.userId)
+    fetchUserDetails();
+  }, [user]);
+
+  if(loading) return (
+    <div>Loading...</div>
+  )
+  return (
+    <div className="bg-slate-100 text-slate-900 relative min-h-screen">
+      {/* Background gradient */}
+      <div className="absolute top-0 left-0 right-0 h-80 bg-gradient-to-r from-indigo-500 to-purple-500 z-0"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)